kubehub/urls.py

```diff
@@ -9,6 +10,10 @@
     url(r'^remove$', crud_views.cloud_provider_remove, name='cloud_provider_remove'),
     url(r'^edit$', crud_views.cloud_provider_edit, name='cloud_provider_edit'),
     url(r'^cluster/create/$', kubespray_deploy.kubespray_deploy, name='cluster_create'),
+    url(r'^VM/group/list$', VmCrudView.vm_groups_list, name='virtual_machines_group_list'),
+    url(r'^VM/group/add$', VmCrudView.vm_group_add, name='virtual_machines_group_add'),
```

Comment:
why VmCrudView not VmGroupCrudView? and make VM uppercase it's abbreviation

Reply:
changed  VmCrudView to vm_group_crud_view (used lowercase becouse its file name) 7f1c04a